include/nlohmann/json.hpp

```diff
@@ -953,7 +953,7 @@ class basic_json
         /// constructor for rvalue strings
         json_value(string_t&& value)
         {
-            string = create<string_t>(std::move(value));
+            string = create<string_t>(std::forward < string_t&& > (value));
```

Comment:
`std::move` was correct here (and in all the other places in this PR) since `value` is not a forwarding reference, but an rvalue reference.

The behaviour is the same in this case, but is quite confusing.